reference/CMakeLists.txt

```diff
@@ -64,6 +64,7 @@ target_sources(
         solver/common_gmres_kernels.cpp
         solver/idr_kernels.cpp
         solver/ir_kernels.cpp
+        solver/lobpcg_kernels.cpp
```

Comment:
I think we will put it into eigensolver

Reply:
I was wondering about that, since it would mean adding a whole new subdirectory, right? Is there some sort of rule for agreeing on things like that among the developers?

Reply:
Moved things into new eigensolver directory in 447005c2cb3bb4fc4d1761ec35a350ec0c5a618e.